lib/src/parallax.dart

```diff
@@ -235,20 +240,33 @@ enum LayerFill { height, width, none }
 /// layer moves with different velocities to give an effect of depth.
 class Parallax {
   Vector2 baseVelocity;
+  Vector2 _size = Vector2.zero();
+  Rect _clipRect;
   final List<ParallaxLayer> layers;
 
   Parallax(
-    this.layers, {
+    this.layers,
+    Vector2 size, {
     Vector2 baseVelocity,
-  }) : baseVelocity = baseVelocity ?? Vector2.zero();
+  })  : assert(layers != null),
+        assert(size != null) {
+    this.baseVelocity = baseVelocity ?? Vector2.zero();
+    resize(size);
```

Comment:
Can't we use the `onGameResize`, and have size as an optional parameter?

Reply:
I don't think so, `onGameResize` is a `BaseGame` thing right? This is the plain feature with no relation with FCS

Reply:
But I think we can use `game.size` on the loader extensions as a default value on the size parameter!

Reply:
Ah right, thought this was in the component. That makes sense!